### PR TITLE
contacts: prune unmanaged "share" flow, rename to "groups"

### PR DESCRIPTION
**Pull request by matildepark**
- `~contacts` -> `~groups`
- Renamed the title in the tile and on the HTML page accordingly.
- Removed the dummy `Share` button in the root identity view
- Removed `/~/default` from the group sidebar
- Rename "Root Identity" to "Your Identity"

### CODE REVIEW

**Comment by matildepark** [comment]
Included the built file by accident, standby ...

**Comment by matildepark** [comment]
OK, pulling that compiled file is a gigantic pain in the ass because of the simultaneous renaming. I wonder if I should just `gulp bundle-prod` it to bring the diff down?

**Comment by loganallenc** [comment]
I'd recommend not moving the files from app/contacts to app/groups. The files are still being served to the browser by contact-view.

**Comment by loganallenc** [comment]
Everything else about this change looks good.

**Comment by matildepark** [comment]
You mean in pkg/arvo? And still mounting at `~groups` in Landscape?

**Comment by loganallenc** [comment]
Correct.

File: pkg/interface/groups/src/js/components/lib/add-contact.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ export class AddScreen extends Component {
     return (
       <div className="h-100 w-100 flex flex-column overflow-y-scroll">
         <div className="w-100 dn-m dn-l dn-xl inter pt1 pb6 pl3 pt3 f8">
-          <Link to={"/~contacts" + props.path}>{"⟵ All Contacts"}</Link>
+          <Link to={"/~groups" + props.path}>{"⟵ All Contacts"}</Link>
```
Comment (loganallenc) [inline pkg/interface/groups/src/js/components/lib/add-contact.js]: All Groups :)

Comment (matildepark) [reply pkg/interface/groups/src/js/components/lib/add-contact.js]: All Groups is the first sidebar. This mobile nav takes you back to the second one, of the contacts in the group.

Comment (loganallenc) [reply pkg/interface/groups/src/js/components/lib/add-contact.js]: kk

**Pull request merged**